### PR TITLE
added an additional paragraph about computer usage and cloudflare

### PR DESCRIPTION
**Pull request by bmuller**

### CODE REVIEW

**Review by artkay — APPROVED** [review]
👍🏻

**Review by noamsauerutley — APPROVED** [review]
👍

**Review by nbgoss81 — APPROVED** [review]
Looks good

**Review by shoosya — APPROVED** [review]

**Comment by shoosya** [comment]
👍

**Review by RickDT — APPROVED** [review]

**Review by nonrational — APPROVED** [review]

**Comment by nonrational** [comment]
image

way to rub it in, github. 😏

File: security/index.md [hunk 1]
```diff
@@ -60,5 +60,9 @@ All Company hardware should have the Company's device profile installed to enabl
...
+
 ## Network :satellite:
 In general, we operate with a stance of [Zero Trust](https://en.wikipedia.org/wiki/Zero_Trust).  We assume that our internal networks could be compromised and seek to secure resources rather than networks.  Given the nature of our [remote workforce](../work/index.md) and our utilization of cloud services, we cannot assume that resources are located within an enterprise-owned network boundary.
+
+To help reduce networking risks, we utilize [Cloudflare for Teams](https://www.cloudflare.com/teams/).  Everyone should install and utilize the Warp client (instructions can be found [here](https://docs.google.com/document/d/1gd_BvZug6p0SD9j77njP3fSa33nyM3cNOMAbqJ5C7fA/edit)).
```
Comment (nonrational) [inline security/index.md:68]: will these instructions move to HDYEE eventually?

Comment (bmuller) [reply security/index.md:68]: They're not specific to engineers or the product team (so maybe not a good fit for HDYEE) - so for now I just have the doc in the "All Hands" shared drive

**Pull request merged**